web/src/components/navigation/mainNav.js

```diff
@@ -2,13 +2,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'gatsby';
 
-export default function MainNav({ navItems }) {
+export default function MainNav({ navItems, subNavItems }) {
     return (
-        <ul className="flex gap-3 justify-end uppercase mt-16 mr-24">
+        <ul className="flex font-semibold gap-3 justify-end mt-16 mr-24 text-white uppercase">
             {navItems.map((item) => {
                 return (
-                    <li key={item.link}>
-                        <Link to={`${item.link}`}>{item.name}</Link>
+                    <li key={item.link} className="group">
+                        <Link
+                            className="hover-list group-hover:block"
+                            to={`${item.link}`}
+                        >
+                            {item.name}
+                        </Link>
+                        <ul>
+                            {subNavItems.map((subitem) => {
+                                return (
+                                    <li key={subitem.link}>
+                                        <Link
+                                            className={`hover-list-link group-hover:block text-brand-1 group-hover:bg-${item.sublinkcolor}`}
+                                            to={`${subitem.subnav}`}
+                                        >
+                                            {subitem.sublink}
+                                        </Link>
+                                    </li>
+                                );
+                            })}
+                        </ul>
```

Comment:
@kcbreuil there needs to be some kind of boolean check here to see if this top level nav item has sub items.

`if (item.subnav) {
    // do map() operation here
  }`

Reply:
@asasmith Yeah I was just working all this out still, check it now :)